README.md

```diff
@@ -72,44 +70,97 @@ how to change monitor address ?
 ## Worker
 
 ```go
-// Limit number of concurrent jobs execution.
-// Use worker.Unlimited (0) if you want no limitation.
-w := worker.New(worker.OneByOne)
-w.ErrHandler = func(e error) {
-	log.Println(e)
+import (
```

Comment:
This is not example. Examples are under `/examples` directory. If you need to add more example or fix those fix it there.
This is just a code sample. Stick with the format as it is, just fix things that needs to be fixed.